.github/actions/sccache/action.yml

```diff
@@ -5,22 +5,30 @@ runs:
   steps:
     - if: runner.os == 'Linux' && runner.arch == 'X64'
       run: |
-        echo "SCCACHE_ARCH=x86_64-unknown-linux-musl" >> $GITHUB_ENV
+          echo "SCCACHE_ARCH=x86_64-unknown-linux-musl" >> $GITHUB_ENV
+          echo "SCCACHE_VERSION=v0.3.3" >> $GITHUB_ENV
       shell: bash
 
     - if: runner.os == 'macOS' && runner.arch == 'X64'
       run: |
         echo "SCCACHE_ARCH=x86_64-apple-darwin" >> $GITHUB_ENV
+        echo "SCCACHE_VERSION=v0.3.3" >> $GITHUB_ENV
       shell: bash
 
     - if: runner.os == 'macOS' && runner.arch == 'ARM64'
       run: |
         echo "SCCACHE_ARCH=aarch64-apple-darwin" >> $GITHUB_ENV
+        echo "SCCACHE_VERSION=v0.3.3" >> $GITHUB_ENV
+      shell: bash
+
+    - if: runner.os == 'Linux' &&  runner.arch == 'ARM64'
+      run: |
+        echo "SCCACHE_ARCH=aarch64-unknown-linux-musl" >> $GITHUB_ENV
+        echo "SCCACHE_VERSION=v0.4.0-pre.5" >> $GITHUB_ENV
```

Comment:
Is there a published version that is the same between all versions of sccache? I worry that mixed versions on different hosts could cause weird random failures.

Reply:
just updated sccache to v.0.5.0 across all systems, and builds are completing successfully